src/main/comments.js

```diff
@@ -976,6 +976,10 @@ function printTrailingComment(commentPath, print, options) {
   }
   const isBlock = util.isBlockComment(comment);
 
+  // Workaround for comments that are in the superClass definition ( extends ...)
```

Comment:
Is this workaround ok? 
Feels dirty to me but there was no other way to get the parentNode of the parentNode